contrib/sync_namespaces_jobs.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+import argparse
+import os
+
+import yaml
+
+DST_DIR = "/tmp/tron-servdir"
+SRC_DIR = "/nail/etc/services/tron/prod"
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='Creating namespaces and jobs configuration from Tron prod'
```

Comment:
If the source is an argument, this description should be more general than "Tron prod".